lib/asm-parser.js

```diff
@@ -30,6 +30,8 @@ class AsmParser extends AsmRegex {
     constructor(compilerProps) {
         super();
 
+        this.commentOnly = /^\s*(((#|@|;|\/\/).*)|(\/\*.*\*\/))$/;
```

Comment:
See my comment below about construction: I'd prefer not poking about in the inside of objects if possible

Reply:
Done.